integrations/firehydrant/CHANGELOG.md

```diff
@@ -13,416 +13,358 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - Upgraded integration dependencies (#1)
 
-
 # Port_Ocean 0.1.58 (2024-07-31)
 
 ### Improvements
 
+- Added _target='blank' attribute to spec links to open a new browser tab instead of the current browser. (#58)
```

Comment:
wrong version in changelog, also why removed the space separating from each version it supposed to be two lines